Catalog.DataAccess/Repositories/CategoryRepository.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Catalog.DataAccess.DTO;
+using Catalog.DataAccess.Interfaces;
+using Catalog.DataAccess.Mappers;
+using Microsoft.EntityFrameworkCore;
+using ORM.Context;
+using ORM.Entities;
+
+namespace Catalog.DataAccess.Repositories
+{
+    public class CategoryRepository : ICategoryRepository
+    {
+        private readonly CatalogContext _dbContext;
+
+        public CategoryRepository(CatalogContext dbContext)
+        {
+            _dbContext = dbContext ?? throw new ArgumentNullException(nameof(dbContext));
+        }
+
+        public void Add(CategoryDal entity)
```

Comment:
In case if you use `async`, please add `Async` suffix to method names

```suggestion
        public async Task AddAsync(CategoryDal entity)
```